src/types/mod.rs

```diff
@@ -1,5 +1,6 @@
 pub mod account_id;
 pub mod api_key;
+pub mod bool;
```

Comment:
update `interactive-clap` to 0.2.4 and remove this type.

Reply:
Done